packages/react-button/src/components/Button/useButton.ts

```diff
@@ -22,6 +22,7 @@ export const useButton = (props: ButtonProps, ref: React.Ref<HTMLElement>, defau
       icon: { as: 'span' },
       // Non-slot props
       size: 'medium',
+      type: 'button',
```

Comment:
Might be worth adding a comment that this is because type="submit" by default, to avoid somebody just removing it later 😆

Reply:
Good idea! I've added the comment.